AES.Web/Controllers/ApplicationController.cs

```diff
@@ -0,0 +1,130 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+using System.Web.Mvc;
+using AES.Web.Models;
+using AES.Web.Authorization;
+
+namespace AES.Web.Controllers
+{
+    public class ApplicationController : Controller
+    {
+        // GET: Application
+        public ActionResult Profile()
+        {
+            var x = FillProfileData();
+            return View(x);
+
+            //return View((ProfileViewModel)TempData["NewProfile"] ?? new ProfileViewModel());
+        }
+
+        [HttpPost]
+        [ValidateAntiForgeryToken]
+        public ActionResult Profile(ProfileViewModel info)
+        {
+            return ValidateRedirect(info, "NewProfile", "Availability");
+        }
+
+       
+        // Temporary hardcoded model
+        public ProfileViewModel FillProfileData()
```

Comment:
Move to end. Keep like code together, and miscellaneous stuff together.
Also, this should be private.